pkg/config/configuration.go

```diff
@@ -562,6 +566,11 @@ func isActionAllowed(action string) bool {
 	return strings.EqualFold(action, AllowAccess)
 }
 
+func getKeyForAppID(appID, namespace string) string {
+	key := appID + "_" + namespace
```

Comment:
We had many discussions about id separators. Can we use "||" here just like we use for state?

Reply:
fixed